amazon/ion/ion_py_objects.py

```diff
@@ -437,19 +423,19 @@ def to_event(self, event_type, field_name=None, in_struct=False, depth=None):
 
 
 class IonPyList_new(list):
-    ion_annotations = ()
-    ion_type = IonType.LIST
-
     __name__ = 'IonPyList_new'
     __qualname__ = 'IonPyList_new'
+    ion_type = IonType.LIST
 
-    def __init__(self, ion_type, value, annotations=()):
+    def __init__(self, ion_type=IonType.LIST, value=None, annotations=()):
+        if value is None:
+            value = []
         super().__init__(value)
         self.ion_annotations = annotations
         # it's possible to be Sexp
         self.ion_type = ion_type
```

Comment:
the constructor still can control ion_type as it's possible to be `Sexp`.

Reply:
Could we call this IonPySequence or something, then? Or does this just need to conform to the existing naming?

Reply:
Yep, It won't break any current use case since we can call it any name we want (E.g., we are calling it an unofficial name with the `_new` suffix). Since the original list and Sexp are called IonPyList, this might affect some write use case (Not sure, maybe for the read use case, people just need to use IonPyList as a native list so they don't need to worry about its class name?).


Or maybe we can redeclare it to ionPyList in `simple_types.py` first, then fully deprecate it when we make breaking changes in the future. 
```
ionPyList = ionPySequence
```